app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -225,7 +228,7 @@ public static void init(Context context) {
             final TelemetryScheduler scheduler = new JobSchedulerTelemetryScheduler();
 
             TelemetryHolder.set(new Telemetry(configuration, storage, client, scheduler)
-                    .addPingBuilder(new TelemetryCorePingBuilder(configuration))
+                    .addPingBuilder(new CustomCorePingBuilder(configuration))
```

Comment:
Does it mean the capture count will be logged for every telemetry event? or?

Reply:
The count will be logged as core ping, daily basis, not as a UI event ping.